modules/ROOT/pages/introduction-monitoring-metrics.adoc

```diff
@@ -105,54 +109,63 @@ However, the MicroProfile Metrics feature doesn't provide metrics about Open Lib
 
 If you enable both the MicroProfile Metrics and Performance Monitoring features, then the MXBeans for monitoring and the `/metrics` REST endpoint are activated.
 In this case, metrics for Open Liberty server components are exposed through both interfaces.
-The MicroProfile Metrics feature version 2.3 and later automatically enables the Performance Monitoring feature.
+The MicroProfile Metrics feature versions 2.3 and later automatically enable the Performance Monitoring feature.
 
-==  Integration with Micrometer
+[#micrometer]
+== Integration with Micrometer
 
 The feature:mpMetrics-5.0[display=MicroProfile Metrics 5.0] feature integrates with the Micrometer metrics technology to transfer metric data to multiple third-party monitoring systems. Micrometer enables you to instrument your code in a standardized way to visualize metrics results in the monitoring system of your choice. Micrometer can connect to 18 different third-party metric monitoring systems. For a full list of metric monitoring systems, see the https://micrometer.io/docs[Micrometer metrics documentation].  
 
-You can integrate Open Liberty with Micrometer by providing the Micrometer core library, Micrometer registry implementations, and the related dependencies to the Open Liberty runtime. This configuration strategy does not provide access to the Micrometer metrics API in the application.
+You can integrate Open Liberty with Micrometer by completing the following steps.
```

Comment:
You can integrate Open Liberty with one or more of the monitoring systems that work with Micrometer by completing the following steps.